eng/jacoco-test-coverage/pom.xml

```diff
@@ -163,6 +163,11 @@
       <artifactId>azure-storage-file-datalake</artifactId>
       <version>12.2.0-beta.1</version> <!-- {x-version-update;com.azure:azure-storage-file-datalake;current} -->
     </dependency>
+    <dependency>
```

Comment:
If this is an internal only implementation do we need coverage numbers?

Reply:
I have plans to add a few more tests as I create PRs for the features that depend on this, so I'll bump the coverage numbers and see where we're at then. For now, I think this is the level of testing that we've been doing in .NET and Python as well